### PR TITLE
Feat/new commands

### PR DESCRIPTION
**Pull request by vcabrales**
### What does this PR do?
Adding new commands (jaja, chupala, vv, hpvv)
Updating existing command (fuck -> fucku)
Adding an updated command (fuck)

### How do I test it?
npm install
npm run start:dev and can send the commands mentioned in the PR

### Background context
When someone say something very funny you can express how funny it was
If someone needs to express a compliment to a friend
If someone needs to express distress
If someone makes a poorly choice
If someone is surprised by a comment/action

### CODE REVIEW

File: app/commands.js [hunk 1]
```diff
@@ -74,4 +74,13 @@ export default (app) => {
     return `${giphyUrl}search?api_key=${giphyToken}&q=${encodeURIComponent(queryString)}&limit=${limit}`;
   }
 
+  app.command('jaja', (ctx) => {
+    ctx.reply(staticResponses['jaja']);
+  });
+  
+  app.command('chupala', (ctx) => {
+    const index = Math.floor((Math.random() * staticResponses.chupala.length-1) + 1);
```
Comment (ghost) [inline app/commands.js]: Random number could be generated from `getRandomNum` function if it receives a limit

**Review by ghost — CHANGES_REQUESTED** [review]

File: dist/commands.js [hunk 2]
```diff
@@ -94,4 +94,13 @@ exports.default = function (app) {
   var getSearchGiphyURL = function getSearchGiphyURL(queryString) {
     return giphyUrl + 'search?api_key=' + giphyToken + '&q=' + encodeURIComponent(queryString) + '&limit=' + limit;
   };
+
+  app.command('jaja', function (ctx) {
+    ctx.reply(_staticResponses2.default['jaja']);
+  });
+
+  app.command('chupala', function (ctx) {
+    var index = Math.floor(Math.random() * _staticResponses2.default.chupala.length - 1 + 1);
```
Comment (jtinoco22) [inline dist/commands.js]: - 1 + 1 != 0 ?? WTF

Comment (ghost) [inline dist/commands.js]: are you missing a parenthesis? any way you will change this to getRandomNum

File: app/commands.js [hunk 3]
```diff
@@ -66,21 +66,33 @@ export default (app) => {
...
 
-  app.command('jaja', (ctx) => {
+  app.command('jaja', ctx => {
     ctx.reply(staticResponses['jaja']);
   });
   
-  app.command('chupala', (ctx) => {
-    const index = Math.floor((Math.random() * staticResponses.chupala.length-1) + 1);
+  app.command('chupala', ctx => {
+    const index = getRandomNum(staticResponses.chupala.length-1 + 1);
```
Comment (ghost) [inline app/commands.js]: -1 + 1 == 0,  I think you can remove this.

File: app/commands.js [hunk 4]
```diff
@@ -32,27 +32,27 @@ export default (app) => {
   });
 
   app.command('buenas', ctx => {
-    const index = Math.floor((Math.random() * staticResponses.buenas.length-1) + 1);
+    const index = getRandomNum(staticResponses.buenas.length-1 + 1);
```
Comment (ghost) [inline app/commands.js]: Same here (length - 1 + 1) == length

File: app/commands.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ import httpStatus from 'http-status-codes';
 
 const giphyUrl = process.env.giphy_url || config.get('giphy_url');
 const giphyToken = process.env.giphy_token || config.get('giphy_token');
-const limit = 5;
+const limit = 25;
```
Comment (ghost) [inline app/commands.js]: You can change this to @bokvilla suggestion GIPHY_API_LIMIT

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**